terraform.tf.j2

```diff
@@ -160,3 +173,18 @@ data "terraform_remote_state" "management_dns" {
     dynamodb_table = "remote_state_locks"
   }
 }
+
+data "terraform_remote_state" "mgmt_ca" {
+  backend   = "s3"
+  workspace = local.management_account[local.environment]
+
+  config = {
+    bucket         = "{{terraform.state_file_bucket}}"
+    key            = "terraform/dataworks/aws-certificate-authority.tfstate"
+    region         = "{{terraform.state_file_region}}"
+    encrypt        = true
+    kms_key_id     = "arn:aws:kms:{{terraform.state_file_region}}:{{terraform.state_file_account}}:key/{{terraform.state_file_kms_key}}"
+    dynamodb_table = "remote_state_locks"
+
+  }
+}
```

Comment:
New line required pls :)

Reply:
Actually - please can you put this further up the file with the rest of the remote states.

Reply:
done, it is actually just below the rest of the remote states.